src/components/problem/styledProblem.js

```diff
@@ -4,23 +4,58 @@ import PercentImg from '../../images/plastic-percents.svg'
 export const ProblemWrapper = styled.section`
   padding: 120px 0;
   background: #1C383C;
+
+  @media( max-width: 1023px ) {
+    padding: 80px 0;
+  }
```

Comment:
At 375px the padding at the top should only be 40px

Reply:
Nice catch